meta-lmp-base/recipes-bsp/alsa-state/alsa-state.bbappend

```diff
@@ -0,0 +1,25 @@
+FILESEXTRAPATHS:prepend := "${THISDIR}/${PN}:"
+
+# ostree-based distros should not commit any files into /var
+# Thus, install a vendor-specific asound.state file into
+# /usr/lib/alsa/ for ostree images.
+VAR_STATEDIR  = "${localstatedir}/lib/alsa"
+INIT_STATEDIR = "${@bb.utils.contains('IMAGE_CLASSES', 'image_types_ostree', "${prefix}", "${localstatedir}", d)}/lib/alsa"
```

Comment:
I would just make this the default behavior in lmp, with and without ostree.

It is a good change that is not really depending on ostree, but it was caused by the expectations from it.

Reply:
Ok, fixing.